src/components/pages/home/newton.svelte

```diff
@@ -281,6 +281,8 @@
       />
       <div bind:this={refs[i]} class="absolute aspect-square h-auto w-1/4">
         <img
+          width="167"
```

Comment:
why is this needed?

Reply:
<img width="721" alt="image" src="https://github.com/significa/significa.co/assets/98809371/54bae1aa-80d0-438b-99b1-3458edc62719">